web/src/javascript/gameLeaderboard.js

```diff
@@ -23,3 +27,31 @@ export function drawLeaderboard(state) {
   const old = document.getElementById("leaderboard-body");
   leaderboard.replaceChild(new_tbody, old);
 }
+
+function focusOnPlayer(id, state){
+    var cellToFocus = findBiggestCell(findPlayerWithId(id, state.players).cells);
+    if(cellToFocus != null){
+        setFocusScreen(cellToFocus.position);
+    }
+}
+
+function findPlayerWithId(id, players){
+  for(const player of players) {
+    if(player.id == id){
+      return player;
+    }
+  }
+}
+
+function findBiggestCell(cells){
```

Comment:
does not work. As i understand : (const max = Math.max.apply(cells.map(cell => cell.radius));) returns the radius  directly so the second line just returns -1 cause it cant find it